### PR TITLE
Fix typos and add initial helm support

### PR DESCRIPTION
**Pull request by eranchetz**
@avivl this relates to #9 , we need to enable gh-pages for this repo on the docs folder after the merge.

### CODE REVIEW

File: docs/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1beta2
```
Comment (avivl) [inline docs/templates/deployment.yaml]: Why beta?

File: docs/templates/kuberbs-crd.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (avivl) [inline docs/templates/kuberbs-crd.yaml]: beta?

File: docs/templates/kuberbs-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+              configMapKeyRef:
+                key: "KUBERBS_APPKEY"
+                name: "kuberbs-config"
+          - name: "KUBERBS_DEBUG"
+            valueFrom:
+              configMapKeyRef:
+                key: "KUBERBS_DEBUG"
+                name: "kuberbs-config"
+
+          - name: GOOGLE_APPLICATION_CREDENTIALS
```
Comment (avivl) [inline docs/templates/kuberbs-deployment.yaml]: What the should be done on none GKE cluster?

File: docs/templates/kuberbs-crd.yaml (hunk 2 above)
Comment (eranchetz) [reply docs/templates/kuberbs-crd.yaml]: took it from : https://github.com/doitintl/kuberbs/blob/476e31acaa776756110664baddc105a959f79782/deploy/kuberbs-crd.yaml#L1

Comment (avivl) [reply docs/templates/kuberbs-crd.yaml]: ok

File: docs/templates/kuberbs-deployment.yaml (hunk 3 above)
Comment (eranchetz) [reply docs/templates/kuberbs-deployment.yaml]: @avivl it's not clear from the readme, if we use a non gke cluster, should line 49 be removed?

Comment (avivl) [reply docs/templates/kuberbs-deployment.yaml]: If we don't use GKE we don't need this line

**Pull request merged**